### ISSUE TITLE
[Mutating Webhook] With environment variable VAULT_TOKEN: vault:login VAULT_ADDR is unset

### ISSUE THREAD

**Issue by ichasco-heytrade** [banzaicloud/bank-vaults#1263]
Hi,

I am using the feature `vault:login` it works but it unsets the variable `VAULT_ADDR` for the container

I set these variables 

```
env:
  - name: ENVIRONMENT
    value: dev
  - name: VAULT_ADDR
    value: https://vault.example.com
  - name: VAULT_STAGE
    value: development
  - name: VAULT_TOKEN
    value: vault:login
```

and when I do a `printenv` I get all of them except `VAULT_ADDR`

Webhook version: `1.11.2`

Thanks!

**Comment by bonifaido** [comment]
This is totally valid, we need to pass these (and probably VAULT_CERT and other client) variables as well when using `vault:login`.

**Comment by bonifaido** [comment]
In the meantime, you can skip sanitizing environment variables out with by setting `VAULT_ENV_PASSTHROUGH=VAULT_ADDR`

**Comment by ichasco-heytrade** [comment]
Thanks @bonifaido. With `VAULT_ENV_PASSTHROUGH=VAULT_ADDR` works.

**Comment by bonifaido** [comment]
Great! Also, a fixe has been shipped in #1265 .
